src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -640,6 +642,20 @@ class RsErrorAnnotator : RsAnnotatorBase(), HighlightRangeExtension {
         }
     }
 
+    // Detect E0267, E0268: break/continue used outside of loop
+    private fun checkParentIsLooplike(holder: AnnotationHolder, expr: RsLabelReferenceOwner) {
```

Comment:
Let's also rename method somehow because it's not about loop, it's about incorrect place of `break`/`continue` expressions. And at least `break` can be placed inside block expression